R/derive_extreme_event.R

```diff
@@ -199,9 +319,17 @@
 #'   )
 #' )
 #'
-#' # Derive confirmed best overall response (using event_joined())
-#' # CR - complete response, PR - partial response, SD - stable disease
-#' # NE - not evaluable, PD - progressive disease
+#' @caption A more complex example: Confirmed Best Overall Response
+#' @info The final example showcases a use of `derive_extreme_event()`
+#'   to calculate the Confirmed Best Overall Response (CBOR) in an
+#'   ADRS dataset. This example builds on all the previous ones and
```

Comment:
```suggestion
#'   `ADRS` dataset. This example builds on all the previous ones and
```
Maybe also mention something like "as commonly used in oncology clinical trials"?

Reply:
@manciniedoardo there's a couple more cases of this later in the example bullets that also need to be updated